components/gitpod-protocol/src/installation-admin-protocol.ts

```diff
@@ -23,6 +23,11 @@ export interface InstallationAdmin {
     settings: InstallationAdminSettings;
 }
 
+export interface Data {
+    installationAdmin: InstallationAdmin
+    totalUsers: number
```

Comment:
As we add more telemetry data, We could continue adding them here thereby separating it from the `InstallationAdmin` (which integrates into the database).

Reply:
Same goes with the new `Data` struct for the golang side of things in `installation-telemetry` workspace